src/app/containers/ConsentBanner/Canonical/logic/README.md

```diff
@@ -0,0 +1,53 @@
+# Consent Banner Utilities
+
+```js
+import consentBannerUtilities from '.';
+
+const {
+    runInitial,
+    privacyOnAllow,
+    privacyOnReject,
+    cookieOnAllow,
+    cookieOnReject,
+  } = consentBannerUtilities({ setShowPrivacyBanner, setShowCookieBanner });
```

Comment:
Not keen on this API. Why can't it be:

```js
import { runInitial, privacyOnAllow, privacyOnReject, cookieOnAllow, cookieOnReject } from '.';
```

...?

Reply:
Because `setShowPrivacyBanner, setShowCookieBanner` are functions defined by the user. They are called so the user can write their own logic to show and hide the banners

Reply:
Hmmm still seems kinda complicated, vs just passing those functions as callbacks.

```js
import { privacyOnAllow } from '.';

const setShowPrivacyBanner = () { /* bla */ };
privacyOnAllow(setShowPrivacyBanner);
```

Reply:
Discussed IRL, Drew has reasons. It's to do with making this logic easier to consume as the front-end will pass these to an `onClick` React param and we can't pass a param to the `onClick` value without putting that inside an anonymous function, and in doing so we'd have to complicate other areas of the codebase.